pkgs/tools/inputmethods/ibus-engines/ibus-typing-booster/wrapper.nix

```diff
@@ -1,14 +1,10 @@
 { typing-booster, symlinkJoin, hunspellDicts, lib, makeWrapper
-, langs ? [ "de-de" "en-us" "es-es" "fr-moderne" "it-it" "sv-se" "sv-fi" ]
+, langs ? [ "de-de" "en-gb-ise" "en-us" "es-es" "fr-moderne" "it-it" "sv-se" "sv-fi" ]
 }:
 
 let
 
-  hunspellDirs = with lib; makeSearchPath ":" (flatten (forEach langs (lang: [
-    "${hunspellDicts.${lang}}/share/hunspell"
-    "${hunspellDicts.${lang}}/share/myspell"
-    "${hunspellDicts.${lang}}/share/myspell/dicts"
-  ])));
+  hunspellDirs = lib.makeSearchPath "share/hunspell" (lib.attrVals langs hunspellDicts);
```

Comment:
The `myspell` directories are [symlinks](https://github.com/NixOS/nixpkgs/blob/0f5996b524c91677891a432cc99c7567c7c402b1/pkgs/development/libraries/hunspell/dictionaries.nix#L17-L20) to `hunspell`.